app/components/navbar.tsx

```diff
@@ -38,20 +38,9 @@ const Navbar: React.FC = () => {
 	const additionalLinks = [
 		{href:"https://github.com/goinvo/staffplan-next-app", label: 'Open Source' },
 		{ href:"mailto:staffplan@goinvo.com",label: 'Feedback' },
-		{href:"https://staffplan.com/settings", label: 'Settings' }
-
+		{href:"https://staffplan.com/settings", label: 'Settings' },
+		{href:"https://staffplan.com/sign_out", label: 'Sign Out' }
```

Comment:
If you want this to work locally too (which it probably should?) you'll have to make the protocol/domain env specific. Locally it should go to `http://localhost:3000/sign_out`.